### PR TITLE
Update filter pruning documentation

### PR DESCRIPTION
**Pull request by evgeniya-egupova**

### CODE REVIEW

File: beta/examples/tensorflow/classification/README.md [hunk 1]
```diff
@@ -115,5 +115,5 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
...
 |ResNet50|INT8 w:sym,per-tensor a:sym,per-tensor + Sparsity 50% (Magnitude)|ImageNet|74.46|[resnet50_imagenet_magnitude_sparsity_int8.json](configs/sparsity_quantization/resnet50_imagenet_magnitude_sparsity_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_int8_w_sym_t_a_sym_t_sparsity_50.tar.gz)|
-|ResNet50|Filter Pruning 40%|ImageNet|74.98|[resnet50_imagenet_pruning_geometric_median.json](configs/pruning/resnet50_imagenet_pruning_geometric_median.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_pruning_40.tar.gz)|
+|ResNet50|Filter Pruning 40%, 4.29/7.75 GFLOPS, 15.8/25.5 MParams|ImageNet|74.98|[resnet50_imagenet_pruning_geometric_median.json](configs/pruning/resnet50_imagenet_pruning_geometric_median.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_pruning_40.tar.gz)|
```
Comment (alexsu52) [inline beta/examples/tensorflow/classification/README.md:118]: Could you directly provide information on how the MParams and GFlops are reduced? What do you think about creating a separate table for pruned models with additional columns such as `GFlops reduction` and `MParams reduction` where you can provide a percentage of pruning and absolute numbers.

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: nncf/tensorflow/README.md [hunk 2]
```diff
@@ -66,14 +67,14 @@ compression_ctrl.export_model("compressed_model.pb", save_format='frozen_graph')
 For a quick start with NNCF-powered compression, you can also try the sample scripts, each of them provides a basic training pipeline for Image Classification and Object Detection correspondingly.
 
 To run the samples please refer to the corresponding tutorials:
-- [Image Classification sample](examples/tensorflow/classification/README.md)
-- [Object Detection sample](examples/tensorflow/object_detection/README.md)
-- [Instance Segmentation sample](examples/tensorflow/segmentation/README.md)
+- [Image Classification sample](/examples/tensorflow/classification/README.md)
```
Comment (vshampor) [inline nncf/tensorflow/README.md:70]: Wrong link target

**Review by vshampor — CHANGES_REQUESTED** [review]

File: nncf/tensorflow/README.md (hunk 2 above)
Comment (evgeniya-egupova) [reply nncf/tensorflow/README.md:70]: So are you suggesting to use `../examples/tensorflow/classification/README.md` or what? These links work fine in github.

Comment (vshampor) [reply nncf/tensorflow/README.md:70]: @evgeniya-egupova, right, use the `../` format.

Comment (evgeniya-egupova) [reply nncf/tensorflow/README.md]: Done

**Review by alexsu52 — APPROVED** [review]

**Review by vshampor — APPROVED** [review]

File: docs/compression_algorithms/Pruning.md [hunk 3]
```diff
@@ -2,10 +2,11 @@
 
 #### Filter pruning    
 
-Filter pruning algorithm zeros output filters in Convolutional layers based on some filter importance criterion  (filters with smaller importance are pruned).     
-The framework contains three such criteria: `L1`, `L2` norm, and `Geometric Median`. Also, different schemes of pruning application are presented by different schedulers.    
+Filter pruning algorithm zeros output filters in Convolutional layers based on some filter importance criterion (filters with smaller importance are pruned).
+Not all Convolution layers in the model can be pruned. Such layers are determined by the model architecture.
```
Comment (AlexKoff88) [inline docs/compression_algorithms/Pruning.md:6]: I would add "Such layers are determined by the model architecture automatically as well as cross-layer dependencies that impose constraints on pruning filters".

File: examples/torch/classification/README.md [hunk 4]
```diff
@@ -99,17 +99,17 @@ As an example of NNCF convolution binarization capabilities, you may use the con
...
-|ResNet-34|None|ImageNet|73.26|[resnet34_imagenet.json](configs/pruning/resnet34_imagenet.json)|-|
-|ResNet-34|Filter pruning, 40%, geometric median criterion|ImageNet|72.72|[resnet34_pruning_geometric_median.json](configs/pruning/resnet34_pruning_geometric_median.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/models/develop/resnet34_imagenet_filter_pruning_geomean.pth)|
-|GoogLeNet|None|ImageNet|69.72|[googlenet_imagenet.json](configs/pruning/googlenet_imagenet.json)|-|
-|GoogLeNet|Filter pruning, 40%, geometric median criterion|ImageNet|68.89|[googlenet_pruning_geometric_median.json](configs/pruning/googlenet_pruning_geometric_median.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/models/develop/googlenet_imagenet_filter_pruning_geomean.pth)|
+|Model|Compression algorithm|Dataset|PyTorch compressed accuracy|GFLOPS|MParams|NNCF config file|PyTorch Checkpoint|
```
Comment (AlexKoff88) [inline examples/torch/classification/README.md:102]: Can we change from "PyTorch compressed accuracy" to "Accuracy (Drop) %", and fill the cells, like "75.4 (-0.9)"

File: docs/compression_algorithms/Pruning.md (hunk 3 above)
Comment (evgeniya-egupova) [reply docs/compression_algorithms/Pruning.md]: Done

File: examples/torch/classification/README.md (hunk 4 above)
Comment (evgeniya-egupova) [reply examples/torch/classification/README.md]: Done for pruning results

Comment (alexsu52) [reply examples/torch/classification/README.md]: @evgeniya-egupova Could you update all the result tables with the suggested method?

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build

**Comment by alexsu52** [comment]
Jenkins please retry a build

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build

**Comment by evgeniya-egupova** [comment]
Jenkins please stop all builds

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please stop all builds

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please stop all builds

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by alexsu52** [comment]
Jenkins please retry a build

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build

**Comment by evgeniya-egupova** [comment]
Jenkins please stop all builds

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build

**Comment by alexsu52** [comment]
Jenkins please retry a build

**Comment by alexsu52** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
Jenkins please retry a build
